### PR TITLE
Revise chapter 01

### PR DESCRIPTION
**Pull request by kepper**
This provides a first draft for #80. Especially the last chapter "samples encodings and tools" is merely more than a stub, but I think it's sufficient to start with.

### CODE REVIEW

File: _guidelines-v4/01-introduction/01-about/01-designprinciples.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+version: "v4"
+title: "MEI Design Principles"
+sectionid: "designprinciples"
+---
+
+This section of the Guidelines defines principles and criteria for designing, developing, and maintaining an XML-based encoding scheme for music notation documents.
+
+##### Definitions and Parameters
+
+A music notation document is one that contains music notation; that is, any one of a number of "visual analogues of musical sound, either as a record of sound heard or imagined, or as a set of visual instructions for performers." (Ian D. Bent, et al. "Notation." Grove Music Online. Oxford Music Online. 25 May 2010. [http://www.oxfordmusiconline.com/subscriber/article/grove/music/20114](http://www.oxfordmusiconline.com/subscriber/article/grove/music/20114){:target='\_blank'}.)
```
Comment (bwbohl) [inline _guidelines-v4/01-introduction/01-about/01-designprinciples.md]: maybe use typographic quotation marks ”…”?

File: _guidelines-v4/01-introduction/01-about/01-designprinciples.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+With respect to metadata, MEI recognizes the close relationship between the metadata content found in the MEI header and that of catalog records, authority records, and finding aids, and it provides for the use of an encoding equivalency attribute for MEI elements corresponding to fields in other metadata standards.
+
+To ensure broad international and multi-repertoire application of MEI, existing musical terminology was used in building the data model where practical. When appropriate, a more neutral terminology was used to facilitate sharing of concepts and thus stressing the commonalities between different repertoires. Finally, extensive use of attributes and clearly-defined classification mechanisms in the schema permits the refinement of element meanings with specific musical, geographic, or temporal contexts.
```
Comment (bwbohl) [inline _guidelines-v4/01-introduction/01-about/01-designprinciples.md]: …element meanings ~~with~~within specific musical, geographic, or temporal contexts.

File: _guidelines-v4/01-introduction/01-about/01-designprinciples.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+##### Control and Maintenance
+
+The Music Encoding Initiative Community has given itself [By-laws](https://music-encoding.org/community/mei-by-laws.html){:target='\_blank'}, which regulate all essential properties and procedures. The community elects a [Board](https://music-encoding.org/community/mei-board.html){:target='\_blank'}, which in turn governs and represents the community. The Board consists of nine elected members, with three seats standing for election for three year terms each year. Everyone registered to the [MEI-L](https://music-encoding.org/community/community-contacts.html){:target='\_blank'} mailing list is eligible to vote for the Board.
+
+In addition to the Board, there is a [Technical team](https://music-encoding.org/community/technical-team.html){:target='\_blank'}, which is open for anyone interested to work on the maintenance and improvement of MEI itself. The Technical team will assist Interest Groups and other interested community members in an advisory capacity on how to further develop MEI for both existing and new fields of application.
```
Comment (bwbohl) [inline _guidelines-v4/01-introduction/01-about/01-designprinciples.md]: Technical ~~t~~Team

File: _guidelines-v4/01-introduction/01-about/02-acknowledgments.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+version: "v4"
+title: "Acknowledgments"
+sectionid: "acknowledgments"
+---
+
+Many institutions and individuals assisted in the preparation of these Guidelines and in the overall development of the Music Encoding Initiative framework.
```
Comment (bwbohl) [inline _guidelines-v4/01-introduction/01-about/02-acknowledgments.md]: …framework and community.

File: _guidelines-v4/01-introduction/01-about/02-acknowledgments.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+Grateful acknowledgment is given to the following institutions for their generous contributions: the Akademie der Wissenschaften und der Literatur (AdW) in Mainz for serving as hosting institution for the MEI Community, and the National Endowment for the Humanities (NEH) and the Deutsche Forschungsgemeinschaft (DFG) for their joint financial support of the MEI project in its early stages. We thank several institutions that hosted Music Encoding Conferences or other MEI-related meetings in the past: The AdW Mainz, the University of Virginia Library, the Biblioteca Umanistica of the Università degli Studi Firenze, McGill University Montréal, the Centre d’études supérieures de la Renaissance Tours, the Maryland Institute for Technology in the Humanities (MITH) in College Park, the Oxford e-Research Centre, the Universität Paderborn and the Österreichische Akademie der Wissenschaften Wien. We also thank all other institutions that allow their researchers to invest time into both the community and the encoding framework. It is their interest that makes MEI an incredible platform for interchange and scholarly progress.
```
Comment (bwbohl) [inline _guidelines-v4/01-introduction/01-about/02-acknowledgments.md]: besides ÖAW: Mozarteum Foundation Salzburg, University of Vienna

File: _guidelines-v4/01-introduction/01-about/02-acknowledgments.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+The Text Encoding Initiative is also owed a special debt of gratitude. In addition to providing much of the inspiration for MEI, the TEI organization supplied funding for the MEI Technical Group in its efforts to adopt ODD. The editors of these Guidelines are grateful for those of the TEI, which provided a stellar exemplar and from which we have borrowed shamelessly.
```
Comment (bwbohl) [inline _guidelines-v4/01-introduction/01-about/02-acknowledgments.md]: …grateful ~~for~~ to those of the TEI, ~~which~~ who…

File: _guidelines-v4/01-introduction/02-basicConcepts/01-musicaldomains.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+---
+version: "v4"
+title: "Musical Domains"
+sectionid: "musicalDomains"
+---
+
+The term "music" has many different notions, ranging from audible sounds over written performance instructions or transcriptions of such events to conceptual rulesets that establish different theories of what music is, and what is allowed in music. In 1965, Milton Babbitt distinguished between _graphemic_, _acoustic_ and _auditory_ aspects of music (Babbitt, Milton: _The Use of Computers in Musicological Research_, in: _Perspectives of New Music_ 3/2 (1965), p. 74–83).
```
Comment (bwbohl) [inline _guidelines-v4/01-introduction/02-basicConcepts/01-musicaldomains.md]: typographic quotation marks “…”

Babbitt: if you want the precise page: 76

File: _guidelines-v4/01-introduction/02-basicConcepts/03-meiprofiles.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+mei-all_anyStart
+: This profile of MEI is even less practical than _mei-all_. It includes all of _mei-all_, but extends it even further so that it allows any MEI element as root of conforming MEI instances. In regular MEI, the only allowed starting elements are {% include link elem="mei"%}, {% include link elem="meiHead"%}, {% include link elem="music"%} and {% include link elem="meiCorpus"%}. The sole purpose of this customization is to simplify validation at tutorial sessions and other educational purposes. It should not be used in production.
+
+mei-CMN
+: For most users, this will be the best starting point into music encoding with MEI. The _mei-CMN_ customization targets at documents that use _Common Western Music Notation_. The specific rules for that notation are specified in chapter {% include link id="cmn" %}, even though other chapters of these Guidelines apply as well.
```
Comment (bwbohl) [inline _guidelines-v4/01-introduction/02-basicConcepts/03-meiprofiles.md]: delete first sentence in order to maintain neutrality

File: _guidelines-v4/01-introduction/02-basicConcepts/03-meiprofiles.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+
+mei-Mensural
+: For documents written in _Mensural Notation_ (both black and white), MEI offers the _mei-Mensural_ customization. The specific rules for that notation are specified in chapter {% include link id="mensural" %}, even though other chapters of these Guidelines apply as well.
+
+mei-Neumes
+: This profile allows to encode medieval Neume Notation with MEI. The specific rules for that notation are specified in chapter {% include link id="neumes" %}, even though other chapters of these Guidelines apply as well. Please note that the _mei-Neumes_ profile has undergone significant changes from MEI version 3 to version 4.
```
Comment (bwbohl) [inline _guidelines-v4/01-introduction/02-basicConcepts/03-meiprofiles.md]: render italic:
_Neume Notation_

File: _guidelines-v4/01-introduction/02-basicConcepts/04-meicustomization.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
+---
+version: "v4"
+title: "Customizing MEI"
+sectionid: "meicustomization"
+---
+
+In production, it is best to use a customized version of MEI, restricted to the very needs of a project. Such a custom schema will guide the encoders and will help to ensure consistency and data quality throughout the project's files. A customization typically provides a subset of MEI's encoding models, with only one solution for any given situation being allowed. The customization will help to reflect the scope of a project into its data. Larger editorial projects like _Complete Works_ editions typically use _Editorial Guidelines_ (german: _Editionsrichtlinien_) for the same purposes: (internal) quality control and (external) documentation. In that sense, MEI customizations may serve as Editorial Guidelines in digital form.
```
Comment (bwbohl) [inline _guidelines-v4/01-introduction/02-basicConcepts/04-meicustomization.md]: …throughout ~~the~~a project's files.

Comment (bwbohl) [reply _guidelines-v4/01-introduction/02-basicConcepts/04-meicustomization.md]: what do you want to express with the sentence:
The customization will help to reflect the scope of a project into its data.

File: _guidelines-v4/01-introduction/01-about/01-designprinciples.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+
+As an encoded representation of one or more music notation documents, an MEI file may be employed as a surrogate for the original materials.
+
+Although the encoding scheme does not define or prescribe intellectual content for music notation documents, it does define content designation and is intended to be used with available data content standards. MEI identifies the essential data elements within music notation documents and establishes codes and conventions necessary for capturing and distinguishing information within those elements for future action or manipulation. While there are a few elements that ought to appear in any MEI document, various intellectual, technical, and economic factors influence the level of detail of analysis and encoding actually undertaken. Taking this into consideration, the encoding scheme is designed with a minimum of required elements and allows for progressively more detailed levels of description as desired.
```
[the remark this answers is not in the document]
Comment (kepper) [reply _guidelines-v4/01-introduction/01-about/01-designprinciples.md]: ask Perry – that's old text, which I'm not going to change now. We can look into that language later…

File: _guidelines-v4/01-introduction/01-about/02-acknowledgments.md (hunk 6 above)
Comment (kepper) [reply _guidelines-v4/01-introduction/01-about/02-acknowledgments.md]: this refers to the TEI Guidelines, not the editors of the TEI Guidelines…

File: _guidelines-v4/01-introduction/01-about/01-designprinciples.md [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+
+##### Definitions and Parameters
+
+A music notation document is one that contains music notation; that is, any one of a number of "visual analogues of musical sound, either as a record of sound heard or imagined, or as a set of visual instructions for performers." (Ian D. Bent, et al. "Notation." Grove Music Online. Oxford Music Online. 25 May 2010. [http://www.oxfordmusiconline.com/subscriber/article/grove/music/20114](http://www.oxfordmusiconline.com/subscriber/article/grove/music/20114){:target='\_blank'}.)
+
+The encoding scheme permits both the creation of new music notation documents and the conversion of existing ones from print and other electronic formats. However, conversion of existing documents may require revisions in content or rearrangement of information.
+
+##### General Principles
+
+No prima facia distinction is made between a primary source of music notation, such as an autograph or published score, and a secondary source, such as a scholarly edition based on one or more primary sources. The tag set encompasses both, and the encoder must choose the elements and attributes most appropriate in each case.
```
[the remark this answers is not in the document]
Comment (kepper) [reply _guidelines-v4/01-introduction/01-about/01-designprinciples.md]: done

File: _guidelines-v4/01-introduction/01-about/01-designprinciples.md [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+
+With respect to metadata, MEI recognizes the close relationship between the metadata content found in the MEI header and that of catalog records, authority records, and finding aids, and it provides for the use of an encoding equivalency attribute for MEI elements corresponding to fields in other metadata standards.
```
[the remark this answers is not in the document]
Comment (kepper) [reply _guidelines-v4/01-introduction/01-about/01-designprinciples.md]: done

File: _guidelines-v4/01-introduction/02-basicConcepts/03-meiprofiles.md (hunk 8 above)
Comment (kepper) [reply _guidelines-v4/01-introduction/02-basicConcepts/03-meiprofiles.md]: done

File: _guidelines-v4/01-introduction/02-basicConcepts/04-meicustomization.md (hunk 10 above)
Comment (kepper) [reply _guidelines-v4/01-introduction/02-basicConcepts/04-meicustomization.md]: tried to clarify

File: _guidelines-v4/01-introduction/01-about/01-designprinciples.md [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+
+##### Structural Features
+
+The encoding scheme is based on eXtensible Markup Language (XML), a text-based format for representing structured information. It is expressed as a One Document Does-it-all (ODD) document, referred to as the "Music Encoding (MEI) Guidelines". For more information on ODD, please refer to TEI Guidelines chapter 22: [Documentation Elements](http://www.tei-c.org/release/doc/tei-p5-doc/en/html/TD.html){:target='\_blank'}, chapter 23: [Using the TEI](http://www.tei-c.org/release/doc/tei-p5-doc/en/html/USE.html){:target='\_blank'}, and to the TEI's "[Getting Started with P5 ODDs](https://tei-c.org/guidelines/customization/getting-started-with-p5-odds/){:target='\_blank'}" document.
```
[the remark this answers is not in the document]
Comment (kepper) [reply _guidelines-v4/01-introduction/01-about/01-designprinciples.md]: not going to implement them now – should be checked throughout the Guidelines. Please file a separate issue.

File: _guidelines-v4/01-introduction/01-about/01-designprinciples.md (hunk 2 above)
Comment (kepper) [reply _guidelines-v4/01-introduction/01-about/01-designprinciples.md]: done

File: _guidelines-v4/01-introduction/02-basicConcepts/01-musicaldomains.md (hunk 7 above)
Comment (kepper) [reply _guidelines-v4/01-introduction/02-basicConcepts/01-musicaldomains.md]: done, thanks for the ref

File: _guidelines-v4/01-introduction/02-basicConcepts/03-meiprofiles.md (hunk 9 above)
Comment (kepper) [reply _guidelines-v4/01-introduction/02-basicConcepts/03-meiprofiles.md]: done

File: _guidelines-v4/01-introduction/01-about/01-designprinciples.md (hunk 3 above)
Comment (kepper) [reply _guidelines-v4/01-introduction/01-about/01-designprinciples.md]: done

File: _guidelines-v4/01-introduction/01-about/02-acknowledgments.md (hunk 5 above)
Comment (kepper) [reply _guidelines-v4/01-introduction/01-about/02-acknowledgments.md]: done

File: _guidelines-v4/01-introduction/01-about/02-acknowledgments.md (hunk 4 above)
Comment (kepper) [reply _guidelines-v4/01-introduction/01-about/02-acknowledgments.md]: done (though I'm not sure about this one)

**Comment by kepper** [comment]
after having a brief look, please merge ahead.

File: _guidelines-v4/01-introduction/02-basicConcepts/03-meiprofiles.md [hunk 15]
```diff
@@ -0,0 +1,26 @@
...
+
+mei-all
+: This is the full definition of MEI. It includes all different repertoires, which has certain side effects and enables encoding options that are neither intended nor advocable. For example, in mensural notation music is organized by staves. In contrast, Common Music Notation utilizes measures, which in turn contain staves. These staves have a different meaning here, and are modeled differently in MEI. _mei-all_ mixes those models and thus invites encoding errors. In general, you should almost never use _mei-all_ except for testing purposes.
+
+mei-all_anyStart
+: This profile includes all of _mei-all_, but extends it even further so that it allows any MEI element as root of conforming MEI instances. In regular MEI, the only allowed starting elements are {% include link elem="mei"%}, {% include link elem="meiHead"%}, {% include link elem="music"%} and {% include link elem="meiCorpus"%}. The sole purpose of this customization is to simplify validation at tutorial sessions and other educational purposes. It should not be used in production.
```
Comment (musicEnfanthen) [inline _guidelines-v4/01-introduction/02-basicConcepts/03-meiprofiles.md]: Please consider to move these lines (mei-all and anySart) to the end (after CMN, Mensural and Neumes). If it is not recommended to use it in general, the reader should be confronted with the recommended customizations first.

File: _guidelines-v4/01-introduction/01-about/01-designprinciples.md [hunk 16]
```diff
@@ -0,0 +1,41 @@
...
+
+##### Definitions and Parameters
+
+A music notation document is one that contains music notation; that is, any one of a number of "visual analogues of musical sound, either as a record of sound heard or imagined, or as a set of visual instructions for performers." (Ian D. Bent, et al. "Notation." Grove Music Online. Oxford Music Online. 25 May 2010. [http://www.oxfordmusiconline.com/subscriber/article/grove/music/20114](http://www.oxfordmusiconline.com/subscriber/article/grove/music/20114){:target='\_blank'}.)
+
+The encoding scheme permits both the creation of new music notation documents and the conversion of existing ones from print and other electronic formats. However, conversion of existing documents may require revisions in content or rearrangement of information.
+
+##### General Principles
+
+No prima facie distinction is made between a primary source of music notation, such as an autograph or published score, and a secondary source, such as a scholarly edition based on one or more primary sources. The tag set encompasses both, and the encoder must choose the elements and attributes most appropriate in each case.
```
Comment (musicEnfanthen) [inline _guidelines-v4/01-introduction/01-about/01-designprinciples.md]: Please simplify

File: _guidelines-v4/01-introduction/02-basicConcepts/03-meiprofiles.md [hunk 17]
```diff
@@ -0,0 +1,26 @@
+---
+version: "v4"
+title: "MEI Profiles"
+sectionid: "meiprofiles"
+---
+
+MEI is an encoding framework, not a data format. This means that MEI provides recommendations for encoding music documents, but it depends on the encoder's needs and requirements to which features and solutions are appropriate to the task and should to be used. MEI offers specific models for different notation types and music repertoires, but it is rarely advisable to use them all side by side in one encoding.
+
+In order to use MEI, it is advised to use a restricted version of the schema, which will make it easier both for an encoder and a reader of the encoded files. MEI provides a number of pre-defined _profiles_, which focus on specific uses of MEI while still maintaining a great level of flexibility. For projects that need even better control over their data, it is highly recommended to create a customized version of MEI (see chapter {% include link id="meicustomization" %}). The following customizations are provided with every release of MEI:
```
Comment (musicEnfanthen) [inline _guidelines-v4/01-introduction/02-basicConcepts/03-meiprofiles.md]:
```suggestion
In order to use MEI, it is advised to use a restricted version of the schema, which will make it easier both for an encoder and a reader of the encoded files. MEI provides a number of pre-defined _profiles_, which focus on specific uses of MEI while still maintaining a great level of flexibility. For projects that need even better control over their data, it is highly recommended to create a more specific customized version of MEI (see chapter {% include link id="meicustomization" %}). The following pre-defined _profiles_ are provided with every release of MEI:
```

File: _guidelines-v4/01-introduction/02-basicConcepts/04-meicustomization.md [hunk 18]
```diff
@@ -0,0 +1,13 @@
+---
+version: "v4"
+title: "Customizing MEI"
+sectionid: "meicustomization"
+---
+
+In production, it is best to use a customized version of MEI, restricted to the very needs of a project. Such a custom schema will guide the encoders and will help to ensure consistency and data quality throughout a project's files. A customization typically provides a subset of MEI's encoding models, with only one solution for any given situation being allowed. The customization will help to reflect the scope of a project into its data: Only those aspects of music notation a project is interested in will be allowed, so that the absence of a specific information can not be misunderstood as an oversight of the encoders. Larger editorial projects like _Complete Works_ editions typically use _Editorial Guidelines_ (german: _Editionsrichtlinien_) for the same purposes: (internal) quality control and (external) documentation. In that sense, MEI customizations may serve as Editorial Guidelines in digital form.
```
Comment (musicEnfanthen) [inline _guidelines-v4/01-introduction/02-basicConcepts/04-meicustomization.md]: Please clarify a little more about the connection of these customizations to the aforementioned MEI profiles. I.e. "customized version of MEI" means actually "even more specific, customized derivation based on a pre-built MEI _profile_"?

File: _guidelines-v4/01-introduction/01-about/01-designprinciples.md (hunk 1 above)
Comment (musicEnfanthen) [inline _guidelines-v4/01-introduction/01-about/01-designprinciples.md]: There are also haptic or other-modal equivalents than just "visual" (cf. Braille). Bents "definition" is quite narrow in this sense.

**Comment by kepper** [comment]
Thanks @musicEnfanthen for the helpful feedback. Would you mind to have another look now?

File: _guidelines-v4/01-introduction/02-basicConcepts/03-meiprofiles.md (hunk 15 above)
Comment (musicEnfanthen) [reply _guidelines-v4/01-introduction/02-basicConcepts/03-meiprofiles.md]: Thank you

File: _guidelines-v4/01-introduction/02-basicConcepts/03-meiprofiles.md (hunk 17 above)
Comment (musicEnfanthen) [reply _guidelines-v4/01-introduction/02-basicConcepts/03-meiprofiles.md]: Thanks

File: _guidelines-v4/01-introduction/01-about/01-designprinciples.md (hunk 1 above)
Comment (musicEnfanthen) [reply _guidelines-v4/01-introduction/01-about/01-designprinciples.md]: Thanks for the addition

**Comment by musicEnfanthen** [comment]
Thanks for your great work in writing this up!

**Comment by musicEnfanthen** [comment]
Just waiting for @bwbohl to approve his requested changes then it could be merged.

**Comment by kepper** [comment]
Thanks. However, credits also need to go to @pe-ro, who wrote significant parts of this as an introduction to our former tag library and guidelines, back in 2010-12 ;-)

File: _guidelines-v4/01-introduction/02-basicConcepts/01-musicaldomains.md [hunk 19]
```diff
@@ -0,0 +1,27 @@
...
+
+Even though the technical implementation of MEI prioritizes the visual domain to some degree, this does not mean that any given encoding has to provide visual information. MEI takes no assumption on what data is required: While an OMR project (_optical music recognition_) may generate strictly visually oriented data only, another project focussed on audio transcriptions may generate gestural data only. A third project could integrate both approaches.
+
+In order to avoid ambiguous encodings, MEI is very strict and specific on the scope of its individual markup elements. For an encoder, the suffixes mentioned above provide clear hints on which domain is addressed by specific markup: Many attributes carry a suffixed _.log_ (logical), _.ges_ (gestural), _.vis_ (visual), or _.log_ (logical) in their name. In addition, the internal structure of MEI heavily relies on those different domains. When customizing MEI (see chapter {% include link id="meicustomization" %}), it is possible to turn off either visual or gestural domain encoding completely. That way, MEI allows to address the four most eminent musical domains specifically and independent of each other.
```
Comment (bwbohl) [inline _guidelines-v4/01-introduction/02-basicConcepts/01-musicaldomains.md]: double mention of .log (logical) domain attributes

File: _guidelines-v4/01-introduction/02-basicConcepts/04-meicustomization.md [hunk 20]
```diff
@@ -0,0 +1,13 @@
...
+
+MEI is implemented in ODD. ODD, or _One Document Does-it-all_, is another XML-based markup language developed and maintained by the TEI. TEI's documentation for ODD can be found in the TEI Guidelines chapter 22: [Documentation Elements](http://www.tei-c.org/release/doc/tei-p5-doc/en/html/TD.html){:target='\_blank'}, chapter 23: [Using the TEI](http://www.tei-c.org/release/doc/tei-p5-doc/en/html/USE.html){:target='\_blank'}, and the "[Getting Started with P5 ODDs](https://tei-c.org/guidelines/customization/getting-started-with-p5-odds/){:target='\_blank'}" document.
```
Comment (bwbohl) [inline _guidelines-v4/01-introduction/02-basicConcepts/04-meicustomization.md]: documents

File: _guidelines-v4/01-introduction/02-basicConcepts/01-musicaldomains.md (hunk 19 above)
Comment (bwbohl) [inline _guidelines-v4/01-introduction/02-basicConcepts/01-musicaldomains.md]: double mention of .log (logical) domain attributes

File: _guidelines-v4/01-introduction/02-basicConcepts/04-meicustomization.md (hunk 20 above)
Comment (kepper) [reply _guidelines-v4/01-introduction/02-basicConcepts/04-meicustomization.md]: not sure what you mean here – the linked "Getting Started with P5 ODDs" is one document, isn't it? Could you clarify?

**Comment by musicEnfanthen** [comment]
> However, credits also need to go to @pe-ro, who wrote significant parts of this as an introduction to our former tag library and guidelines, back in 2010-12 ;-)

I see. Time to thank @pe-ro once more for his foresight, thank you!

**Comment by kepper** [comment]
thanks everyone – glad to have this out of the door now :-)

**Pull request merged**